### ISSUE TITLE
Java VM options should be configured IDE independently in gradle build

### ISSUE THREAD

**Issue by Katharsas** [FAForever/downlords-faf-client#1546]
Move them from .idea/runConfigurations to build.gradle (or separate file that is imported in build.gradle)

Example gradle task configuration:
```gradle
bootRun {
  jvmArgs = ["-Dprism.dirtyopts=false",
      "-Xms128m", "-Xmx512m",
      "-XX:MinHeapFreeRatio=15", "-XX:MaxHeapFreeRatio=33",
      "-XX:+HeapDumpOnOutOfMemoryError", "-XX:+UseStringDeduplication",
      "-javaagent:webview-patch/build/libs/webview-patch.jar"]
}
```
example above did work for me in Eclipse to automatically launch with specified parameters, needs evaluation by somebody with IntelliJ.

**Comment by 1-alex98** [comment]
Well i think that would not work for intellij I think intellij needs manual set parameters

**Comment by 1-alex98** [comment]
or we change to run config to use gradle task

**Comment by iczero** [comment]
yeah i think idea won't pick up args in gradle :(

**Comment by 1-alex98** [comment]
Depending on what the run config is yes

**Comment by Sheikah45** [comment]
Closed by #2346
